cmd/download.go

```diff
@@ -19,10 +22,20 @@ func addDownload(cmd *cobra.Command) {
 	downloadCmd.Flags().StringArray("tag", []string{}, "Only download the resources with the given tag")
 	downloadCmd.Flags().StringArray("notag", []string{}, "Only download the resources without the given tag")
 	downloadCmd.Flags().String("perm", "", "Optional permissions for the downloaded files (e.g. '644')")
+	downloadCmd.Flags().BoolP("verbose", "v", false, "Enable verbose output")
 	cmd.AddCommand(downloadCmd)
 }
 
 func runFetch(cmd *cobra.Command, args []string) error {
+	logLevel, _ := cmd.Flags().GetString("log-level")
```

Comment:
I think adding an explicit way to set the log level isn't a bad idea... but it should be done in this PR. Let's keep this PR focused on *one* thing please: adding the optimization to skip the downloading if the resource is there and its integrity matches.

Reply:
You don't want to comment them out. They need to be *removed*. (And moved to a different PR if you want).